parts/common/components.json

```diff
@@ -785,6 +785,11 @@
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/kube-proxy",
           "latestVersion": "v1.31.5",
           "previousLatestVersion": "v1.31.4"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/kube-proxy",
```

Comment:
while we are caching one more version, is it appropriate to remove the oldest one?